src/index.html

```diff
@@ -12,6 +12,202 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="fieldset">
```

Comment:
There should be no blank lines between parent and child blocks, fix it everywhere
```suggestion
    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
      <fieldset class="fieldset">
```